segway/gmtk/params.py

```diff
@@ -317,6 +419,31 @@ def __str__(self) -> str:
         return " ".join([self.mean, self.covar])
 
 
+class MissingFeatureDiagGaussianMC(MC, object):
+    """
+    Attributes:
+        component_type = "COMPONENT_TYPE_MISSING_FEATURE_SCALED_DIAG_GAUSSIAN"
+        mean: str: name of Mean object associated to this MC
+        covar: str: name of Covar obejct associated to this MC
+    """
+    def __init__(self, mean: str, covar: str):
+        """
+        Initialize a single MissingFeatureDiagGaussianMC object.
+        :param mean: name of Mean object associated to this MC
+        :param covar: name of Covar obejct associated to this MC
+        """
+        # more component types?
+        super().__init__("COMPONENT_TYPE_MISSING_FEATURE_SCALED_DIAG_GAUSSIAN")
+        self.mean = mean
+        self.covar = covar
+
+    def __str__(self) -> str:
+        """
+        Return string representation of this MC object.
+        """
+        return " ".join([self.mean, self.covar, "matrix_weightscale_1x1"])
```

Comment:
Put `matrix_weightscale_1x1` in a class attribute or instance attribute whose name says what it is supposed to be.

Reply:
I'm unsure what `matrix_weightscale_1x1` corresponds to, as the GMTK documentation does not discuss the missing feature Gaussian and the term is not used elsewhere. If you know what this represents and what I should name the attribute, I'll add it as that attribute.